### PR TITLE
uploaded

### PR DESCRIPTION
**Pull request by shadowshot-x**
backend part 1

### CODE REVIEW

**Comment by shadowshot-x** [comment]
this is the intended backend

File: canteen/orders/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Dishes',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('dish_name', models.CharField(max_length=200)),
+                ('price', models.IntegerField()),
+                ('dish_logo', models.CharField(max_length=1000)),
```
Comment (shivam04) [inline canteen/orders/migrations/0001_initial.py:20]: change charField to imagefield 
and define function for the location of image before putting it into db

File: canteen/orders/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                ('cuisines', models.CharField(max_length=200)),
+                ('estimated_time', models.IntegerField()),
```
Comment (shivam04) [inline canteen/orders/migrations/0001_initial.py:22]: what's estimated time ?

File: canteen/orders/models.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.conf import settings
+from django.utils import timezone
+# Create your models here.
+class Dishes(models.Model):
+    dish_name=models.CharField(max_length=200)
+    price=models.IntegerField()
+    dish_logo=models.CharField(max_length=1000)
```
Comment (shivam04) [inline canteen/orders/models.py:8]: As said above _^_

File: canteen/orders/tests.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
```
Comment (shivam04) [inline canteen/orders/tests.py:2]: add unit tests

File: canteen/orders/views.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
+from django.shortcuts import render
+from .models import Dishes
+from django.utils import timezone   
+from django.shortcuts import render, get_object_or_404
+# Create your views here.
+def list_of_dishes(request):
+    return render(request, 'orders/view1.html', {})
```
Comment (shivam04) [inline canteen/orders/views.py:7]: template name should be meaningful

File: canteen/orders/migrations/0001_initial.py (hunk 1 above)
Comment (shadowshot-x) [reply canteen/orders/migrations/0001_initial.py:20]: okkay sir! Noted

File: canteen/orders/migrations/0001_initial.py (hunk 2 above)
Comment (shadowshot-x) [reply canteen/orders/migrations/0001_initial.py:22]: estimated tiem in preparation of the dish

File: canteen/orders/views.py (hunk 5 above)
Comment (shadowshot-x) [reply canteen/orders/views.py:7]: yes, the templates will be added

File: canteen/orders/migrations/0001_initial.py (hunk 2 above)
Comment (shivam04) [reply canteen/orders/migrations/0001_initial.py:22]: instead, use estimated_delivery_time after food is prepared.

**Pull request closed without merging**